update-nft-metadata-in-treasury/README.md

```diff
@@ -0,0 +1,30 @@
+# Update NFT Metadata Using the Supply Key in Treasury Account
+
+This feature allows you to update an NFT's metadata while it's still in the treasury account, using the supply key. Once the NFT is transferred out, its metadata becomes immutable, ensuring security and consistency.
+
+
+This code snippet demonstrates creating an NFT collection, minting an NFT, and updating the NFT's metadata using the supply key.
+
+## How to use this code snippet
+
+1. Rename `.env.example` file to `.env`
+2. Update the values `OPERATOR_ID` and `OPERATOR_KEY` with your credentials
+3. In your terminal, run `npm i` 
+4. Execute the script with `node index.js`
+
+#### Example output
+```bash
+The NFT collection ID is 0.0.4028
+The NFT serial number is 1
+The transaction ID of the token metadata update transaction is 0.0.1297@1724274772.140049451
+The NFT metadata update transaction was SUCCESS
```

Comment:
I suggest modifying the script such that the output prints the value of the metadata of the token before and after the transaction, for example:

```suggestion
The NFT collection ID is 0.0.4028
The NFT serial number is 1
The NFT meta data is: ${INITIAL_METADATA}
The transaction ID of the token metadata update transaction is 0.0.1297@1724274772.140049451
The NFT metadata update transaction was SUCCESS
The NFT meta data is: ${FINAL_METADATA}
```